package NYT.NMetaState.NProto;

import "yt/core/misc/guid.proto";
import "yt/core/rpc/rpc.proto";

////////////////////////////////////////////////////////////////////////////////

message TMutatingExt
{
    extend NRpc.NProto.TRequestHeader
    {
        optional NYT.NProto.TGuid mutation_id = 101;
    }
}

////////////////////////////////////////////////////////////////////////////////
